tailwind.config.ts

```diff
@@ -20,6 +20,9 @@ const config: Config = {
       },
       screens: {
         smr: '480px',
+        customBreakpointHandler: '1169px', // There was a mysterious break point around 1169px in HeaderClient.tsx,
```

Comment:
I feel that this name is a bit long, what about something like `md-lg`?

Reply:
sure, I have renamed it